Using "" may cause a "connection refused" error in some environments.
Using the localhost IP is more reliable.
diff --git a/test/distributed/elastic/test_control_plane.py b/test/distributed/elastic/test_control_plane.py
index ede4e352b04..c34bd4b1919 100644
--- a/test/distributed/elastic/test_control_plane.py
+++ b/test/distributed/elastic/test_control_plane.py
@@ -157,7 +157,7 @@ class WorkerServerTest(TestCase):
 
         from torch._C._distributed_c10d import _WorkerServer
 
-        server = _WorkerServer("", 1234)
+        server = _WorkerServer("127.0.0.1", 1234)
         out = requests.get("http://localhost:1234/handler/")
         self.assertEqual(out.status_code, 200)
 
